mitosheet/src/plugin.tsx

```diff
@@ -319,22 +319,22 @@ function activateMitosheetExtension(
         focus on it, so the user can just starting typing in it!
     */
     app.commands.addKeyBinding({
-        command: 'mitosheet:focus-on-search',
+        command: 'mitosheet:open-search',
         args: {},
         keys: ['Accel F'],
         selector: '.mito-container'
     });
-    app.commands.addCommand('mitosheet:focus-on-search', {
+    app.commands.addCommand('mitosheet:open-search', {
```

Comment:
Where does this command get called?

Reply:
Directly above this code ^^ `app.commands.addKeyBinding({ command: 'mitosheet.open-search' ... })`